next/components/forms/segments/AccountNavBar/AccountNavBar.tsx

```diff
@@ -86,15 +125,86 @@ export const AccountNavBar = ({ className, currentLanguage }: IProps) => {
               </p>
             }
           />
+          <nav className="text-font/75 flex gap-x-8 font-semibold">
+            <div className="text-font/75 flex items-center gap-x-6 font-semibold">
+              {!navHidden ? (
+                <>
+                  <Link href="/" variant="plain">
```

Comment:
please verify accessibility guidelines (I don't have a good reference unfortunately, but search around the web for some baseline for webpage menus) - would at least expect a list element somewhere around here, maybe some aria markers as needed ?

Reply:
in fact, a good baseline might be to use https://react-spectrum.adobe.com/react-aria/useMenu.html - we're trying to move to react-aria everywhere it makes sense. The useMenu seems to handle these kinds of menus well, should take care of accessibility and should let you style as needed.

Same for other menus in this PR.